src/cards/card.test.ts

```diff
@@ -30,6 +30,50 @@ describe('card', () => {
     await CardModel.insertMany(initialCards);
   });
 
+  describe('PUT /cards/:id route', () => {
+    describe('given the request is authorized and data is correct', () => {
+      it('should return with a 200 status code and newly updated flashcard', async () => {
+        const flashcards = await findCards({});
+        const flashcardToUpdate = flashcards[0];
+
+        const response = await api
+          .put(`/cards/${flashcardToUpdate?._id}`)
+          .send(card)
+          .set(supertestConfig);
+
+        const { _id, createdAt, updatedAt, ...flashcardStripped } =
+          response.body;
+
+        expect(response.statusCode).toBe(200);
+        expect(JSON.stringify(card) === JSON.stringify(flashcardStripped)).toBe(
```

Comment:
**nitpick:** Asserting objects using `JSON.stringify` is not the best practice. Have you tried: 

```
expect(card).toEqual(flashcardStripped);
```